site/specs/voice.yml

```diff
@@ -4197,9 +4197,7 @@ paths:
       description: >-
```

Comment:
I can't comment on the line above this, but maybe we should rename the summary to `Get Recording Transcription`?